src/Workspaces/CSharp/Portable/CodeGeneration/CSharpCodeGenerationHelpers.cs

```diff
@@ -262,5 +262,18 @@ public static CodeGenerationDestination GetDestination(SyntaxNode destination)
                 : node;
             return result;
         }
+
+        public static T GetReuseableSyntaxNodeForSymbol<T>(ISymbol symbol, CodeGenerationOptions options) where T : SyntaxNode
```

Comment:
can you add docs that hte node returned will not have a usable parent?

Reply:
Sure